### PR TITLE
chore(lib): Move passport-local-with-otp into force

### PR DESCRIPTION
**Pull request by damassi**
Follow up to https://github.com/artsy/force/pull/9263, which moves `@artsy/passport-local-with-otp` directly into force, so we don't have to manage more libs. 

Tests all pass: 

Screen Shot 2022-01-25 at 2 02 41 PM

### CODE REVIEW

File: src/lib/passport/lib/passport/index.js [hunk 1]
```diff
@@ -2,54 +2,62 @@
...
-const FacebookStrategy = require('passport-facebook').Strategy;
-const AppleStrategy = require('passport-apple');
-const LocalWithOtpStrategy = require('@artsy/passport-local-with-otp').Strategy;
-const callbacks = require('./callbacks');
-const { serialize, deserialize } = require('./serializers');
-const opts = require('../options');
+const passport = require("passport")
+const FacebookStrategy = require("passport-facebook").Strategy
+const AppleStrategy = require("passport-apple")
+const LocalWithOtpStrategy = require("lib/passport-local-with-otp/lib").Strategy
```
Comment (damassi) [inline src/lib/passport/lib/passport/index.js:8]: ☝️ only actual change

**Review by joeyAghion — DISMISSED** [review]

**Comment by damassi** [comment]
No clue why tests aren't passing -- all good locally. Will dig in a bit.

**Comment by joeyAghion** [comment]
Just following up about this @damassi. I tried rerunning the build but the test failure looks repeatable and related.

**Comment by damassi** [comment]
@joeyAghion - sorry, just now seeing this! Will return to this PR on Fri.

**Comment by damassi** [comment]
Tests keep failing in CI and not sure why. Going to hand this off to #product-velocity

**Comment by anandaroop** [comment]
🟢 but worth double-checking via review app https://relocate-passport-with-otp.artsy.net

**Comment by joeyAghion** [comment]
Thanks @anandaroop. After some initial difficulties (relating to cross-app caches), I was able to log in with 2FA-enrolled users and without, and tested a few error scenarios as well.

**Review by joeyAghion — APPROVED** [review]

**Pull request merged**